### DISCUSSION TITLE
How to get the count of unread feeds using Fever API?

### DISCUSSION THREAD

**Discussion by Jorenar** [FreshRSS/FreshRSS#4802] [Q&A] [answered]
I wasn't able to find anything about it in documentation.  
For now I've hacked together the following:
```sh
 curl -s -F "api_key=$FRESHRSS_API_KEY" "$FRESHRSS_URL/api/fever.php?api&unread_item_ids" | sed 's/.*"".*/,,/' | awk -F"," '{print NF-3}'
```

**Comment by Alkarex** [comment, marked as answer]
It does not look like the *Fever API* supports anything better for that. Documentation: http://feedafever.com/api

However, the *Google Reader API* - which FreshRSS also supports - is richer, more powerful, and more efficient. I recommend the Google Reader API for almost every usage.

The Google Reader API has an `unread-count` method returning that information for each feed:

* https://web.archive.org/web/20210126115837/https://blog.martindoms.com/2009/10/16/using-the-google-reader-api-part-2/#unread-count
* https://web.archive.org/web/20120608040458/http://undoc.in/unread-count.html
* https://feedhq.readthedocs.io/en/latest/api/reference.html#unread-count
* https://www.inoreader.com/developers/unread-counts

The entry `'user/-/state/com.google/reading-list'` will contain the total number of unread articles.
